modules/rostests/apitests/com/com_apitest.h

```diff
@@ -200,6 +200,14 @@ DEFINE_GUID(IID_CDefView,                  0x4434ff80, 0xef4c, 0x11ce, 0xae, 0x6
 DEFINE_GUID(IID_FolderItems2,              0xc94f0ad0, 0xf363, 0x11d2, 0xa3, 0x27, 0x00, 0xc0, 0x4f, 0x8e, 0xec, 0x7f);
 DEFINE_GUID(IID_FolderItems3,              0xeaa7c309, 0xbbec, 0x49d5, 0x82, 0x1d, 0x64, 0xd9, 0x66, 0xcb, 0x66, 0x7f);
 
+
+DEFINE_GUID(IID_IQueryContinue,            0x7307055c, 0xb24a, 0x486b, 0x9f, 0x25, 0x16, 0x3e, 0x59, 0x7a, 0x28, 0xa9);
```

Comment:
If they are available in MS's headers, it might make sense to move them to the correct place already.

Reply:
You put them in shobjidl (which is correct) so they do not need to be here.